### ISSUE TITLE
uninstall operator script is hard-coded to uninstall v0.5.0 only

### ISSUE THREAD

**Issue by stleerh** [sustainable-computing-io/kepler-operator#244]
The hack/uninstall-operator.sh script is hard-coded to look for "v0.5.0".  Therefore, if you have any other version of kepler installed, it will not find it, and hence won't be uninstalled.  I am on the v1alpha1 branch.

**Comment by vprashar2929** [comment]
Yes v0.5.0 is hardcoded but if you run `uninstall-operator.sh` on a cluster with version > 0.5.0 installed it shows you which version is currently available
```

   🔆🔆🔆  Resources of Kepler Operator - v0.5.0  🔆🔆🔆
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Error from server (NotFound): clusterserviceversions.operators.coreos.com "kepler-operator.v0.5.0" not found
 🔔 failed to find v0.5.0 of kepler-operator.
──────────────────────────────────────────────────
NAME                     DISPLAY   VERSION   REPLACES                 PHASE
kepler-operator.v0.8.0   Kepler    0.8.0     kepler-operator.v0.7.2   Succeeded
──────────────────────────────────────────────────
 🔔 kepler-operator version found are ☝️
```
Using this you can run `./hack/uninstall-operator.sh --version v0.8.0 --delete` to remove the current available version

**Comment by vprashar2929** [comment]
@stleerh We have updated(#248) the `uninstall-operator.sh` script to automatically find the version of the Kepler-operator that is installed on the cluster. Feel free to give it a go

**Comment by sthaha** [comment]
Closing this as fixed by #248
